Makefile

```diff
@@ -56,3 +56,13 @@ publish:
 clean:
 	sudo $(RM) -r ./artifacts
 	$(RM) $(MAN_PAGES)
+
+alias-latest:
```

Comment:
I don't think you need this target (sorry I lead you astray my mentioning "symlink"), since `VERSION` can be injected (it uses `?=`) and `build-all` respectes that, you could make another target that just builds again to another name:

```makefile
build-all-latest: VERSION=latest
build-all-latest: build-all
```

Then 

```makefile
release: build-all build-all-latest publish
```